Description: Uncomments the default connection directive of etc/keystone.conf.sample
Author: Ghe Rivero <ghe@debian.org>
Forwarded: not-needed

Index: keystone/etc/keystone.conf.sample
===================================================================
--- keystone.orig/etc/keystone.conf.sample	2013-12-16 16:45:17.000000000 +0800
+++ keystone/etc/keystone.conf.sample	2013-12-16 16:48:40.000000000 +0800
@@ -140,7 +140,7 @@
 
 [sql]
 # The SQLAlchemy connection string used to connect to the database
-# connection = sqlite:///keystone.db
+connection = sqlite:////var/lib/keystone/keystone.sqlite
 
 # the timeout before idle sql connections are reaped
 # idle_timeout = 200
